target/linux/bcm53xx/image/Makefile

```diff
@@ -452,4 +452,14 @@ define Device/tplink_archer-c9-v1
 endef
 TARGET_DEVICES += tplink_archer-c9-v1
 
+define Device/iptime_a5004ns
+  DEFICE_VENDOR = ipTIME
```

Comment:
Typo

Reply:
@aparcar i fixed the typo but i'm still having same issue